command/plan.go

```diff
@@ -48,7 +49,15 @@ General Options:
 
   -force-count
     Use the taskgroup count from the Nomad jobfile instead of the count that
-    is currently set in a running job.
+	is currently set in a running job.
+
+  -output-format
+	Specify the format of plan to be emitted. Valid values include JSON, DIFF,
+	and NDJSON. The default is DIFF.
+
+  -output-to
```

Comment:
`render` has a similar config, but it uses the `-out`:
https://github.com/hashicorp/levant/blob/19af3df6794425e08368d1a8e31fb3886390a53c/command/render.go#L50